build.sbt

```diff
@@ -308,8 +308,8 @@ lazy val cli = project
 
             def zipEntries(zipStream: ZipInputStream): Iterator[(ZipEntry, Array[Byte])] =
               new Iterator[(ZipEntry, Array[Byte])] {
-                var nextEntry = Option.empty[ZipEntry]
-                def update() =
+                private var nextEntry = Option.empty[ZipEntry]
+                private def update() =
```

Comment:
Are these two really a problem? The type ascription just above (`: Iterator[(ZipEntry, Array[Byte])]`) should make these two not escape the scope of the anonymous class, nor create a structural type.